src/server/scripts/World/npc_innkeeper.cpp

```diff
@@ -22,16 +23,24 @@ class npc_innkeeper : public CreatureScript
     bool OnGossipHello(Player* player, Creature* creature) override
     {
         if (IsEventActive(HALLOWEEN_EVENTID) && !player->HasAura(SPELL_TRICKED_OR_TREATED))
-            AddGossipItemFor(player, GOSSIP_ICON_CHAT, "Trick or Treat!", GOSSIP_SENDER_MAIN, GOSSIP_ACTION_INFO_DEF + HALLOWEEN_EVENTID);
+        {
+            AddGossipItemFor(player, 342, 0, GOSSIP_SENDER_MAIN, GOSSIP_ACTION_INFO_DEF + HALLOWEEN_EVENTID);
```

Comment:
what is `342` and shall we move it to a constant?

Reply:
The problem is that there are many records for the same phrase, so I don't know which one to use exactly.
That's why I didn't create the constant.

![trick_or_treat](https://user-images.githubusercontent.com/2810187/118016118-65ee1700-b32b-11eb-8cd7-2f9b14880fbc.png)

Reply:
It seems to work with 342. But since there were so many options, I wasn't sure which one to use.

![WoWScrnShot_051221_150827](https://user-images.githubusercontent.com/2810187/118023947-7eaefa80-b334-11eb-9ee7-6bec21128fa9.jpg)